### ISSUE TITLE
Unable to convert CityJSON to glb

### ISSUE THREAD

**Issue by maaattes** [cityjson/cjio#57]
I'm trying to convert CityJSON to glb with cjio version 0.6.
I've tried to convert the sampe dataset [Neighbourhood 'Delfshaven'. Buildings in LoD2](https://3d.bk.tudelft.nl/opendata/cityjson/1.0/3-20-DELFSHAVEN.json)
The command:  
`cjio 3-20-DELFSHAVEN.json export --format glb 3-20-DELFSHAVEN.glb`
produces
```shell
Parsing 3-20-DELFSHAVEN.json
Exporting CityJSON to glb /home/mattes/vmware/share/Daten/LOD2/3-20-DELFSHAVEN.glb
/home/mattes/.local/lib/python3.8/site-packages/cjio/geom_help.py:39: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  if((n==x3).all()):
Traceback (most recent call last):
  File "/home/mattes/.local/bin/cjio", line 11, in <module>
    load_entry_point('cjio==0.6', 'console_scripts', 'cjio')()
  File "/home/mattes/.local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/mattes/.local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/mattes/.local/lib/python3.8/site-packages/click/core.py", line 1290, in invoke
    return _process_result(rv)
  File "/home/mattes/.local/lib/python3.8/site-packages/click/core.py", line 1224, in _process_result
    value = ctx.invoke(self.result_callback, value, **ctx.params)
  File "/home/mattes/.local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/cjio.py", line 103, in process_pipeline
    cm = processor(cm)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/cjio.py", line 266, in processor
    exporter(cm)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/cjio.py", line 164, in exporter
    glb = cm.export2gltf()
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/cityjson.py", line 1562, in export2gltf
    glb = convert.to_glb(self.j)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/convert.py", line 188, in to_glb
    tri = geom_help.triangulate_face(face, vertexlist)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/geom_help.py", line 88, in triangulate_face
    xy = to_2d(p, n)
  File "/home/mattes/.local/lib/python3.8/site-packages/cjio/geom_help.py", line 39, in to_2d
    if((n==x3).all()):
AttributeError: 'bool' object has no attribute 'all'
```

**Comment by balazsdukai** [comment]
Which python version are you using?
Which OS?

**Comment by maaattes** [comment]
> Which python version are you using?
> Which OS?

I'm using Python 3.8.5 on Fedora 32.

**Comment by tichimura** [comment]
hi, @balazsdukai  it seems the function `get_normal_newell` returns `(n, True)`, and triangulate_face is just passed the result to `to_2d`. So, it could not compare them as numpy array.
https://github.com/cityjson/cjio/blob/v0.6.0/cjio/geom_help.py#L66

just changing to return only n with `return (n)` should avoid this error, but hope this help.

**Comment by balazsdukai** [comment]
Fixed by 98a21a1479d0f3d1285583f0f9be5ac345f6c310
